snmp/requirements.in

```diff
@@ -1,7 +1,7 @@
 ipaddress==1.0.22; python_version < '3.0'
 ply==3.10
 pyasn1==0.4.6
```

Comment:
(nit)

> We can't update pyasn1 for now
because of etingof/pyasn1#178. And we can't
update pysnmp to 4.4.10+ because of etingof/pysnmp#329.

Could you inline those comments in the `requirements.in`. That might be helpful.